package.json

```diff
@@ -7,7 +7,7 @@
     "@ledgerhq/hw-transport-webusb": "^6.27.13",
     "@phosphor-icons/react": "^2.0.10",
     "@react-spring/web": "^9.6.1",
-    "@secretkeylabs/xverse-core": "1.9.1",
+    "@secretkeylabs/xverse-core": "^1.9.1-4628cff",
```

Comment:
be careful the ^ can mess with the exact version installed

Reply:
yup, it will not build the local build locally properly, if the carrot is present